src/More/ENDebugMenu.tsx

```diff
@@ -119,55 +103,55 @@ const ENDebugMenu = ({ navigation }: ENDebugMenuProps): JSX.Element => {
       ) : (
         <ScrollView>
           <View style={styles.section}>
-            <DebugMenuListItem
+            {/* <DebugMenuListItem
```

Comment:
Are these menu items going to be useful in any scenario? If not how do we feel about deleting them and probably the native module functions they invoke?

Reply:
plus one, we should remove code if its not being used.  are these something that we want to keep internally for devs? or remove altogether?